bin/index.js

```diff
@@ -248,6 +249,119 @@ module.exports.createWorkflowRequest = function (token_filename, project_id) {
     });
 }
 
+module.exports.createRunRequest = function (token_filename, component_id, project_id) {
+    if (has_id_token(token_filename)) {
+        public_token_header = read_id_token(token_filename);
+    }
+    var payload = {
+        "component_details":{
+            "component_name": "calibration",
```

Comment:
@saifulbkhan How about sending the`component_name` as an argument instead of the `component_id`? The id can be fetched using the name anyway. Plus this will make this function more flexible so you could just send "intomix" as the component name when we need to integrate that app into El-MAVEN.

Just a suggestion, does not need to be done right now.